packages/ui/src/components/va-avatar/VaAvatar.vue

```diff
@@ -49,9 +50,10 @@ export default defineComponent({
     const { getColor } = useColors()
     const colorComputed = computed(() => getColor(props.color))
     const { sizeComputed, fontSizeComputed } = useSize(props, 'VaAvatar')
+    const { textColorComputed } = useTextColor(props.color)
 
     const computedStyle = computed(() => ({
-      color: getColor(props.textColor, 'white'),
+      color: textColorComputed.value,
```

Comment:
Here and in similar situations. If something isn't used before return, it's better to move it to the return object.

```suggestion
      color: useTextColor(props.color),
```

Reply:
> Here and in similar situations. If something isn't used before return, it's better to move it to the return object.

This way you declare computed inside computed which is completely wrong...

Reply:
> > Here and in similar situations. If something isn't used before return, it's better to move it to the return object.
> 
> This way you declare computed inside computed which is completely wrong...

My bad, wrong example - thought it's `return` (as mentioned in my comment). I meaned situations like this:

`
const { textColorComputed } = useTextColor(props.color)

return {
  color: textColorComputed .value
}
`

Reply:
Anyway. I think computedStyle should stay here and depend on textColorComputed.  We have a lot of example of extra consts, I would rather make a separate task.